exonum-java-binding/core/rust/src/handle/mod.rs

```diff
@@ -17,13 +17,9 @@
 //! Wrappers and helper functions around Java pointers. Used for memory management
 //! between native and Java.
 
-// TODO Remove `allow(dead_code)` after [https://jira.bf.local/browse/ECR-910].
-#![allow(dead_code)]
-
 use jni::sys::jlong;
 use jni::JNIEnv;
```

Comment:
NIT: I'd also nest all `use` statements since we once decided to do so when modifying "old" files.